<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Copyright (c) DevSec Studio. All rights reserved.

MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the &quot;Software&quot;), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/

/*-----QWidget-----*/
QWidget
{
	background-color: #292f45;
;
	color: #fff;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: #292f45;
	color: #b9b9bb;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-color: #3D5AFA;
	padding: 10px;

}


QPushButton::hover
{
	

}


QPushButton::pressed
{
	background-color: #ff6b35;
	color: #fff;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #292f45;
	color: #000000;
	border-style: solid;
	border-color: #000000;

}


QToolButton::hover
{
	background-color: #fc7c11;
	color: #000000;
	padding: 2px;
	border-radius: 15px;
	border-color: #fc7c11;

}


QToolButton::pressed
{
	background-color: #fc7c11;
	color: #000000;
	border-style: solid;
	border-width: 2px;

}


/*-----QLineEdit-----*/
QLineEdit{
	background-color: #292f45;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 2px;
	border-top: 0px;
	border-left: 0px;
	border-right: 0px;
	border-color: #b9b9bb;
	padding: 10px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #b9b9bb;
	font-weight: bold;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #f0742f;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #1f2b2b;
    border: 1px solid #f0742f;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #f0742f;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}
</string>
  </property>
  <widget class="QLineEdit" name="sign_email">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>610</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sign_confirmpassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>610</width>
     <height>50</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #3D5AFA;
	color: #fff;
}


QPushButton::hover
{
	background-color: #22348c;
	border-color: #22348c;
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sign_password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>610</width>
     <height>50</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sign_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>610</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sign_firstname">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sign_lastname">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tosignIn_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-----QPushButton-----*/
QPushButton
{
	background-color: #292f45;
	color: #b9b9bb;
	font-weight: bold;
	border-color: #292f45;
	text-decoration:underline;
}

QPushButton::hover
{
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>Already have an Account ?</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
